src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1126,6 +1126,10 @@ protected virtual async Task<bool> SendCallStack(SessionId sessionId, ExecutionC
                         function_name.StartsWith("_mono_wasm_fire_debugger_agent_message", StringComparison.Ordinal) ||
                         function_name.StartsWith("mono_wasm_fire_debugger_agent_message", StringComparison.Ordinal)))
                 {
+                    if (function_name.StartsWith("$func", StringComparison.Ordinal) && int.TryParse(function_name.Remove(0, 5), out var funcId))
```

Comment:
Can we move this condition into `SymbolicateFunctionName` itself, so references to the `"$func"` string are in one place?